src/Autofac.Pooling/RegistrationExtensions.cs

```diff
@@ -0,0 +1,369 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autofac.Builder;
+using Autofac.Core;
+using Autofac.Core.Activators.ProvidedInstance;
+using Autofac.Core.Lifetime;
+using Autofac.Core.Registration;
+using Autofac.Core.Resolving.Pipeline;
+
+namespace Autofac.Pooling
+{
+    /// <summary>
+    /// Provides registration methods for setting up pooled instances.
+    /// </summary>
+    public static class RegistrationExtensions
+    {
+        /// <summary>
+        /// <para>
+        /// Configure the component so that every dependent component or call to Resolve() within a single <see cref="ILifetimeScope"/>
```

Comment:
It's difficult to point a single method, because there are loads of resolve methods, don't want to give people the idea that only certain resolve methods trigger this behaviour.

For background, that text is a direct copy from the `InstancePerMatchingLifetimeScope` summary in Autofac core.

Reply:
I've just removed that reference entirely in favour of regular English.